extensions/webview/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "reactxp-webview",
+  "version": "0.0.1-alpha.1",
+  "description": "Plugin for ReactXP that provides a control that allows the display of an independent web page",
+  "author": "ReactXP Team <reactxp@microsoft.com>",
+  "license": "MIT",
+  "scripts": {
+    "build": "npm run tslint && tsc",
+    "tslint": "tslint -p tsconfig.json -r tslint.json -r ./node_modules/tslint-microsoft-contrib --fix || true"
```

Comment:
nit: the rules directory lookup can be included in tslint.json directly and removed from this command